lib/meilisearch/rails/ms_clean_up_job.rb

```diff
@@ -0,0 +1,19 @@
+module MeiliSearch
+  module Rails
+    class MSCleanUpJob < ::ActiveJob::Base
+      queue_as :meilisearch
+
+      def perform(documents)
+        documents.each do |document|
+          index = MeiliSearch::Rails.client.index(document[:index_uid])
+
+          if document[:synchronous]
```

Comment:
Is it helpful to do the check instead of just calling async or sync. But I changed my mind 😅. Let's let the user decide; we don't know the use cases. Maybe it is too much to wait for a call in certain apps, but indispensable for others...